packages/vscode-extension/src/debug/taskTerminal/baseTaskTerminal.ts

```diff
@@ -44,6 +44,12 @@ export abstract class BaseTaskTerminal implements vscode.Pseudoterminal {
 
   protected async stop(error?: any): Promise<void> {
     if (error) {
+      if (error.message === "Debug session exists") {
```

Comment:
Make the specific string a constant since it's used in two places, to ensure consistency.

Reply:
Updated.